.parcelrc

```diff
@@ -4,7 +4,8 @@
   "transforms": {
     "docs:*.{js,ts,tsx,json}": ["parcel-transformer-docs", "@parcel/transformer-inline"],
     "docs-json:*.{js,ts,tsx,json}": ["parcel-transformer-docs"],
-    "*.mdx": ["parcel-transformer-mdx-docs"]
+    "*.mdx": ["parcel-transformer-mdx-docs"],
+    "*.svg": ["@parcel/transformer-svg-react"]
```

Comment:
Was having trouble importing svgs (kept complaining about missing a transformer), stumbled upon this solution. Lemme know if an alternate route should be taken here